marketplace_builder/views/pages/tutorials/modules/creating-module.liquid

```diff
@@ -81,6 +81,86 @@ query get_pages {
 
 ### Step 3: Create page
 
+#### A not on naming conventions and paths
+
+Configuration files placed in a module are treated a bit differently than regular files.
+To avoid conflicts between the code of a module and regular code (or between modules) a namspacing strategy is used.
+
+##### Namspacing rules
+
+1. assets (JavaScript, CSS, images) - paths for those files are prefixed with *modules/MODULE_NAME*
+
+For example if you have asset in a file called 'app.css' in a module named "admin" to include it you should use
+
+```liquid
+ <link rel="stylesheet" media="screen" href="{{ 'modules/admin/app.css' | asset_url }}">
+```
+
+ The same thing works for JavaScript files, for example:
+
+```liquid
+<script src="{{ 'modules/admin/admin.js' | asset_url }}"></script>
+```
+
+2. Partials
+
+Partials can use full path with the prefix this is a bit longer but allows us to avoid ambiguity
+
+```liquid
+{%- include 'modules/blog/private/views/partials/comments' -%}
+```
+
+3. Layouts
+
+Layouts have the same prefix so when you use them you should do it in the following way
+
+```liquid
+slug: 'admin/settings'
+format: html
+layout_name: 'layouts/modules/admin/settings'
```

Comment:
I think `modules/admin/layouts/settings` would be more consistend with other namespaces

Reply:
yeah